permissions-report/permissions-report.rb

```diff
@@ -63,9 +63,11 @@ def headers(context)
 
 $table_data = []
 
+# TODO: obtain list of org admins to filter from output
+$org_admins = %w(olivergondza jenkinsadmin rtyler kohsuke daniel-beck oleg-nenashev batmat).map(&:downcase)
```

Comment:
the downcase is there to remind the person that is going to solve the TODO :cold_sweat:

Reply:
Ended up implementing the API